CommonWallet/Classes/Modules/Amount/AmountPresenter.swift

```diff
@@ -93,7 +93,13 @@ final class AmountPresenter {
                                                           symbol: selectedAsset.symbol)
         assetSelectionViewModel.canSelect = account.assets.count > 1
 
-        amountInputViewModel = transferViewModelFactory.createAmountViewModel()
+        var optionalAmount: Decimal?
```

Comment:
Why "optional" in the name? It is obvious by the type

Reply:
Just a habit: at some point you might want to write something like if let amount = optionalAmount {}. But it is not the case here. I will fix it.

Reply:
`if let amount = amount {` works perfectly fine in such a case

Reply:
Sure. It works)